pennylane/_qubit_device.py

```diff
@@ -669,20 +672,22 @@ def active_wires(operators):
 
         return Wires.all_wires(list_of_wires)
 
-    def statistics(self, circuit: QuantumScript, shot_range=None, bin_size=None):
+    # pylint: disable=too-many-statements
```

Comment:
Let's add a TODO here to remove this pylint disable when the measurement refactor is complete.

Reply:
We won't be able to remove all the `if`/`elif` statements because we would need to change the signature of all the measurement methods of the `QubitDevice` class. We decided to leave it as it is, because next year we will refactor all the `QubitDevice` anyway.